### PR TITLE
HL-38 | Add Employee model

### PR DESCRIPTION
**Pull request by quyenlq**
## Description :sparkles:
Add Employee Model

### CODE REVIEW

File: backend/benefit/applications/models.py [hunk 1]
```diff
@@ -260,3 +281,71 @@ class Meta:
...
+    application = models.OneToOneField(
+        Application,
+        verbose_name=_("application"),
+        related_name="employee",
+        on_delete=models.CASCADE,
+    )
+
+    first_name = models.CharField(max_length=128, verbose_name=_("first name"))
+    last_name = models.CharField(max_length=128, verbose_name=_("last name"))
+    social_security_number = models.CharField(
```
Comment (haavikko) [inline backend/benefit/applications/models.py:296]: Kesäseteli uses an encrypted field for the SSN, should we do that too?

File: backend/benefit/applications/models.py [hunk 2]
```diff
@@ -260,3 +281,71 @@ class Meta:
...
+        verbose_name=_("working hour"),
+        decimal_places=1,
+        max_digits=4,
+        blank=True,
+        null=True,
+    )
+
+    # TODO: Adding missing file fields or a separate Attachment relationship
+    # power_of_attorney = models.FileField()
+    # collective_bargaining_agreement = models.FieldField()
```
Comment (haavikko) [inline backend/benefit/applications/models.py:343]: I think that the collective_bargaining_agreement would be a freeform text input (not the actual file itself)

File: backend/benefit/applications/tests/factories.py [hunk 3]
```diff
@@ -71,3 +72,25 @@ def bases(self, created, extracted, **kwargs):
...
+    phone_number = factory.Faker("phone_number")
+    email = factory.Faker("email")
+
+    employee_language = factory.Faker(
+        "random_element", elements=[v[0] for v in APPLICATION_LANGUAGE_CHOICES]
+    )
+    job_title = factory.Faker("job")
+    monthly_pay = factory.Faker("random_int", max=5000)
+    monthly_pay = factory.Faker("random_int", max=5000)
+    monthly_pay = factory.Faker("random_int", max=5000)
```
Comment (haavikko) [inline backend/benefit/applications/tests/factories.py:92]: typo?

**Review by haavikko — CHANGES_REQUESTED** [review]
Some small things

File: backend/benefit/applications/tests/factories.py (hunk 3 above)
Comment (quyenlq) [reply backend/benefit/applications/tests/factories.py]: Forgot to change the name, fixed now 😂

File: backend/benefit/applications/models.py (hunk 1 above)
Comment (quyenlq) [reply backend/benefit/applications/models.py]: Good point, this should be encrypted, fixed

File: backend/benefit/applications/models.py (hunk 2 above)
Comment (quyenlq) [reply backend/benefit/applications/models.py]: Ah, right, yeah I thought it was part of the attachments in the third step

**Review by haavikko — APPROVED** [review]
Looks good! Would be useful to add Django admin support for Employee before committing though?

**Review by klemmari1 — APPROVED** [review]
Looks good to me!

**Pull request merged**